google/cloud/automl/unit-tests/cloud_automl_vision_model_tests.yaml

```diff
@@ -0,0 +1,86 @@
+test:
+  suites:
+  - name: Cloud AutoML Vision Model API IT
+    setup:
+    - code: cloud_region = "us-central1"
+      # Read Project ID from Environment Variable
+    - env:
+        what: GOOGLE_PROJECT_ID
+        var: project_id
+    - env:
+        what: MODEL_ID
+        var: model_id
+    - env:
+        what: MODEL_DISPLAY_NAME
+        var: model_display_name
+    cases:
+    - name: List Model
+      spec:
+      - call: 
+          sample: automl_vision_list_models
+          params: 
+            location: { variable: cloud_region }
+            project_id: { variable: project_id }
+            filter: { literal: "" }
```

Comment:
Why is this here?

Reply:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/8956f1741131ea22063466dbc3e74aa1d26144d4/vision/automl/src/test/java/com/google/cloud/vision/samples/automl/ModelApiIT.java#L55-L57

Reply:
I thought I was converting old IT from java-docs-samples

Reply:
Did you mean why there are two different unit tests files ? @beccasaurus